### PR TITLE
Api provides subscriber functionality to create local (Backyard conspiracies) articles

### PR DESCRIPTION
**Pull request by arunbhalli**
**PT-Story:** https://www.pivotaltracker.com/story/show/178092217
### User Story 
``` 
As an API
In order to support the workings of local, community-based article tabs,
I need to provide the client with functionality to display and create articles
``` 
## Changes proposed in this pull request: 
*  subscriber can create backyard article
* Open up CREATE action and route to `api/backyards/` 
* Put that location in the permit
* restricted to Subscribers of course
## What I have learned working on this feature: 
*  How to use the build method for creating an article

### CODE REVIEW

**Review by DavDan1 — APPROVED** [review]

File: app/controllers/api/backyards_controller.rb [hunk 1]
```diff
@@ -11,10 +13,30 @@ def show
...
+
+  def article_params
+    params[:backyardArticle].permit(:title, :theme, :location, body: [])
+  end
+
+  def build_backyard_article
+    backyard_article = current_user.articles.build(article_params)
+    backyard_article['backyard'] = true
+    backyard_article.save
+    backyard_article
```
Comment (1CIM) [inline app/controllers/api/backyards_controller.rb:40]: what dose this do?

**Review by christerforsgren91 — APPROVED** [review]
sweet beans

File: app/controllers/api/backyards_controller.rb (hunk 1 above)
Comment (1CIM) [reply app/controllers/api/backyards_controller.rb:40]: line 40

Comment (lukeperrin10) [reply app/controllers/api/backyards_controller.rb:40]: we have to return the backyard article after we update it. The .save method only returns a boolean. And ruby only returns the latest line of code, so we have to call on the article again.

**Review by LucasKnudsen — APPROVED** [review]

**Pull request merged**